### DISCUSSION TITLE
You may add a prefix to include guard's macro.

### DISCUSSION THREAD

**Discussion by Barenboim** [wfrest/wfrest#36] [General]
Currently:
\#ifndef _HTTPMSG_H_
\#define _HTTPMSG_H_

Macro '_HTTPMSG_H_' is a little bit short and may raise some conflicts. I suggest to add a WFREST prefix:
\#ifndef WFREST_HTTPMSG_H_
\#define WFREST_HTTPMSG_H_

**Comment by chanchann** [comment]
Ok
